tests/test_lru_cache.py

```diff
@@ -67,3 +69,28 @@ def test_lru_object():
         assert SENTINEL is lru.get(c)
 
     lru.cache_info()
+
+
+def test_expire_object():
+    import time
```

Comment:
global import is recommended for stable std modules

Reply:
Can I ask why? I thought local import is better because the `time` module is only used for this test. So I can skip the import for the other tests.

Reply:
There are a few reasons. Generally, eager failure is easier than deferred failure. If a module must exist and work for any environment, the error of top-most inportings will be raised at module importing time. Otherwise it will be raised in deep side of call stacks which can be missed by low-coverage test suites. I prefer not to create local import unless there is a circular dependency or huge performance drop by importing.
In this case, another matter is the advantage. time is a standard and widely used module which must be used in pytest and ring. So the advantage from local import is very low.

Reply:
> If a module must exist and work for any environment, the error of top-most inportings will be raised at module importing time. Otherwise it will be raised in deep side of call stacks which can be missed by low-coverage test suites.

Oh, now I got it. Thanks for the answer!